OWML.Common/IHarmonyHelper.cs

```diff
@@ -6,12 +6,12 @@ namespace OWML.Common
     public interface IHarmonyHelper
     {
         void AddPrefix<T>(string methodName, Type patchType, string patchMethodName);
-        void AddPrefix(MethodInfo methodInfo, Type patchType, string patchMethodName);
+        void AddPrefix(MethodBase methodInfo, Type patchType, string patchMethodName);
         void AddPostfix<T>(string methodName, Type patchType, string patchMethodName);
-        void AddPostfix(MethodInfo methodInfo, Type patchType, string patchMethodName);
+        void AddPostfix(MethodBase methodInfo, Type patchType, string patchMethodName);
         void EmptyMethod<T>(string methodName);
-        void EmptyMethod(MethodInfo methodInfo);
+        void EmptyMethod(MethodBase methodInfo);
         void Transpile<T>(string methodName, Type patchType, string patchMethodName);
-        void Transpile(MethodInfo methodInfo, Type patchType, string patchMethodName);
+        void Transpile(MethodBase methodInfo, Type patchType, string patchMethodName);
```

Comment:
Pretty sure this breaks backwards compatibility. What's the reason for the change?

Reply:
MethodBase is parent of MethodInfo

Reply:
Ah, cool :)